examples/basic-infection/README.md

```diff
@@ -44,36 +44,36 @@ context::add_module(person_property_report);
 context::execute();
 ```
 
-Individuals transition through a typical SIR pattern they where
+Individuals transition through a typical SIR pattern where they
 start off as susceptible (S), are randomly selected to become infected
 (I), and eventually recover (R).
 
 ```mermaid
 flowchart LR
-S(Susceptible) --FoI--> I(Infected) --inf. period--> R(Recovered)
+S(Susceptible) --foi--> I(Infected) --infection_period--> R(Recovered)
```

Comment:
Attempting to make consistent - don't know if it is too wide to format correctly.